packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -18,7 +18,8 @@ import { RPCProtocol } from '../api/rpc-protocol';
 import { Disposable } from '@theia/core/lib/common/disposable';
 import { LogPart, KeysToAnyValues, KeysToKeysToAnyValue } from './types';
 import { CharacterPair, CommentRule, PluginAPIFactory, Plugin } from '../api/plugin-api';
-import { PreferenceSchema } from '@theia/core/lib/browser/preferences';
+// FIXME get rid of browser code in backend
```

Comment:
is there a way to detect/forbid such imports automatically ?

Reply:
no, should not be hard to write custom tslint rule for our conventions